src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier supplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        Ball randomBall = new Ball();
+        final int maxValue = 100;

```

Comment:
Create constant variable instead of method variable, cause according to the task - maximum value never change